File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x;
    do
    {
        x = get_int("Height: "); // user in put
    }
    while ((x < 1) || (x > 8));

    for (int i = 0; i < x ; i++)
    {
        for (int k = 0; k < x; k++)
         
        {
            if (i + k < x - 1) 
            
                printf(" ");//printf space
                
                else  
                
                    printf("#");//printf hash
                    
            
                }
                printf("\n");// printf new line
                   

            }
    
    
    
        }


for(int i = 0; i < 4; i++){
        for(int k =0; k < number; k++ ){
            printf("*");
        }
        printf("\n");
        number = number -2;
        for(int j = 0; j < i; j++){
            printf(" ");
        }
        
       
    }

File: pset6/mario.py
from cs50 import get_int

def main():
    while True:
        n = get_int("Height:")
        if n >= 1 and n <= 8:
            break
    for hash_a in range( n + 1):
      space = n - hash_a 
      print(" " * space, end="")
      print("#" * hash_a, end="")
      print(" ", end=" ")
      print("#" * hash_a)  
main() 